### PR TITLE
Encounters completion stats - Home Visit + Child Scorebaord

### PR DESCRIPTION
**Pull request by anvmn**
#1295 
#1298

### CODE REVIEW

**Comment by anvmn** [comment]
Selection_836

**Comment by anvmn** [comment]
Selection_841

File: server/hedley/modules/custom/hedley_reports/scripts/completion-recalculate-large-datasets.php [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+      // Explicitly unset large variables after use for memory optimization.
+      unset($json_data);
+    }
+
+    $nid = end($ids);
+    // Explicitly unset large variables after use for memory optimization.
+    unset($nodes);
+
+    $processed += 400;
```
Comment (AronNovak) [inline server/hedley/modules/custom/hedley_reports/scripts/completion-recalculate-large-datasets.php:175]:
```suggestion
    $processed += $batch;
```

**Review by AronNovak — APPROVED** [review]

File: server/hedley/modules/custom/hedley_reports/scripts/completion-recalculate-large-datasets.php (hunk 1 above)
Comment (anvmn) [reply server/hedley/modules/custom/hedley_reports/scripts/completion-recalculate-large-datasets.php:175]: ^^ Fixed at a follow up PR - #1300.

**Pull request merged**